FAQ/en.md

```diff
@@ -0,0 +1,61 @@
+# Frequently Asked Questions
+
+## General
+
+### What state is Quaver currently in?
+
+Quaver is currently in the closed Alpha state. In order to use online features of Quaver, you need to sign up for the Alpha.
+
+### Where can I sign up for the Quaver Alpha?
+
+We announce sign-ups before they open, but until then, you will have to wait. Sign-ups are being held during specific time periods, and are not open at all time.
+
+### Can I play without being part of the Quaver Alpha?
+
+Yes, you can download an offline version from Quaver’s [Github page](https://github.com/Quaver/Quaver/releases), However, you won’t be able to experience any of the online features.
+
+### Does this mean I can’t download maps from the Quaver website?
+
+That is correct, however, Adri (a member of the Quaver team) offers an alternate download [here](https://rhythmgamers.net/pack/) in the form of map packs for anyone to access. Keep in mind he doesn’t update those packs very frequently.
+
+### Is there an estimate for the public release date?
+
+We do not have an estimate for the public release date yet.
+
+
+## Gameplay
+
+### What key modes does Quaver support?
+
+Quaver currently only supports 4 keys and 7 keys.
+
+### Can I import maps from other games?
+
+Yes, Quaver currently supports .osu and .sm files and we are working on supporting more file formats.
+
+### I have a lot of maps in an external game, can I import them all at once?
+
+It is possible to import all of your maps at once by linking the game’s “.db” file inside your “quaver.cfg” file, which can be located inside your Quaver folder.
+
+### Can I import my skin from another game?
+
+Yes, you can download a converter by Adri [here](https://rhythmgamers.net/QBC/) and a tutorial can be found [here](https://www.youtube.com/watch?v=pWeLbx48NVI).
+
+### I found a bug, where can I report it?
+
+Any bug reports should be posted on our [GitHub issues](https://github.com/Quaver/Quaver/issues). It doesn't matter how minor you think the issue is. Make sure to post anything you find, but please check if the issue has already been reported to avoid duplication.
+
+### I have a feature request, where can I post it?
+
+Feature requests belong, just like bug reports, on our [GitHub issues](https://github.com/Quaver/Quaver/issues). Simply choose “Feature Request” when creating a new issue!
+
+
+## Troubleshooting
+
+### The game doesn’t launch
+
+Make sure Steam is running, as Steam is required in order to run Quaver. This also applies to the offline build!
+
+### I can’t upload my mapset using the editor
+
+If your mapset fails to upload using the client, you can upload it manually on the [Upload Mapset](/upload/mapset/) page.
```

Comment:
Link is broken

Reply:
Changed the link to a full quavergame.com link